rundetection/rules/osiris_rules.py

```diff
@@ -108,6 +108,9 @@ def _determine_mode(
     def verify(self, job_request: JobRequest) -> None:
         if not self._value:
             return
+        # Added this so we can turn the rule off
+        if self._value is False:
+            return
```

Comment:
This does the same as the previous 2 lines

Reply:
Removed